src/sentry/nodestore/riak/backend.py

```diff
@@ -40,7 +40,7 @@ class RiakNodeStorage(NodeStorage):
 
     def __init__(
         self,
-        nodes,
+        nodes=None,
```

Comment:
Without this change, it was required to specify a value, even though there was intended to be a fallback to DEFAULT_NODES, so this forced local dev to specify options here.